mps-model-adapters/src/main/kotlin/org/modelix/model/mpsadapters/MPSRepositoryAsNode.kt

```diff
@@ -78,34 +81,34 @@ data class MPSRepositoryAsNode(val repository: SRepository) : IDeprecatedNodeDef
     }
 
     override fun getReferenceTarget(link: IReferenceLink): INode? {
-        TODO("Not yet implemented")
+        return null
     }
 
     override fun setReferenceTarget(link: IReferenceLink, target: INode?) {
-        TODO("Not yet implemented")
+        return
     }
 
     override fun setReferenceTarget(role: IReferenceLink, target: INodeReference?) {
-        TODO("Not yet implemented")
+        return
     }
 
     override fun getReferenceTargetRef(role: IReferenceLink): INodeReference? {
-        TODO("Not yet implemented")
+        return null
     }
 
     override fun getPropertyValue(property: IProperty): String? {
-        TODO("Not yet implemented")
+        return null
     }
 
     override fun setPropertyValue(property: IProperty, value: String?) {
         TODO("Not yet implemented")
     }
 
     override fun getPropertyLinks(): List<IProperty> {
-        TODO("Not yet implemented")
+        return emptyList()
     }
 
     override fun getReferenceLinks(): List<IReferenceLink> {
-        TODO("Not yet implemented")
+        return emptyList()
```

Comment:
These changes should probably somehow preserve the information that the methods are actually not implemented using comments?

Reply:
Oh right, then maybe a test case verifying these conditions or some comment would help to avoid this confusion the next time. The implementation locks very much like a stub.